container.go

```diff
@@ -137,6 +138,11 @@ func (c *container) Labels(ctx context.Context) (map[string]string, error) {
 }
 
 func (c *container) SetLabels(ctx context.Context, labels map[string]string) (map[string]string, error) {
+	ctx, span := tracing.StartSpan(ctx, "container.SetLabels", tracing.SpanEventAttribute(
+		tracing.Attribute("container.id", c.id)),
+	)
+	defer span.End()
+	span.SetAttributes(tracing.Attribute("container.id", c.id))
```

Comment:
Does it need to call both SpanEventAttribute and SetAttributes?

Reply:
No, removed `SetAttributes`